tasklist/client/src/Tasks/Task/FormJS/index.tsx

```diff
@@ -169,7 +190,30 @@ const FormJS: React.FC<Props> = ({
             )
             .otherwise(() => null)}
         </TaskDetailsRow>
-        <DetailsFooter>
+        <DetailsFooter
+          className={styles.buttons}
+          status={
+            savingState === 'finished' ? (
+              <SaveSuccessMessage />
+            ) : savingState === 'error' ? (
+              <SaveFailedMessage />
+            ) : undefined
+          }
+        >
+          <SaveButton
+            savingState={savingState}
+            onClick={() => {
+              save(
+                formatFormDataToVariables(
+                  formManagerRef.current
+                    ?.get('form')
+                    ._getSubmitData() as object,
+                ),
```

Comment:
We shouldn't be using `form-js` internal methods like this, part of the epic should have been changing adapting `form-js` and make this public...